tests/integration/test_utils/data_source_config/sql.py

```diff
@@ -71,8 +72,12 @@ def setup(self) -> None:
             name=self.table_name, df=self.data, column_types=self.config.column_types or {}
         )
         extra_table_data = [
-            TableData(name=name, df=df, column_types=self.config.extra_column_types.get(name, {}))
-            for name, df in self.extra_data.items()
+            TableData(
```

Comment:
I think the changes here do the right thing, but they're done in a kinda ugly way. I plan to circle back and do a slight refactor here after this PR merges.